Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ruby:2.2-jessie
 RUN apt-get update && apt-get install
-RUN gem install bundler
+RUN gem install bundler -v 1.17.3
```

Comment:
Why do we want to force this version?

Reply:
The image can't be built with newer versions of bundler, maybe changing this line to `RUN gem install bundler -v '<= 1.17.3'` may be more accurate